watershed/Vanilla/watershed.py

```diff
@@ -125,12 +114,12 @@
 
         # Provide the Pilot Description
         #
-        pdesc = rp.ComputePilotDescription()
+        pdesc           = rp.ComputePilotDescription()
         pdesc.resource  = resource
         pdesc.project   = project
         pdesc.runtime   = walltime
         pdesc.cores     = pilot_cores
-	pdesc.queue 	= queue
+	    pdesc.queue 	= queue
```

Comment:
Why these extra spaces here? The indentation does not look correct. Can you make sure you are using 4 spaces for each indentation level?

Reply:
That's really  odd. It doesn't show up as that on my computer :\. I'll try to find out what's going on

Reply:
Okay I fixed it